packages/grafbase-sdk/README.md

```diff
@@ -344,3 +344,28 @@ Introspecting the connector namespace to the schema happens with the `datasource
 g.datasource(stripe, { namespace: 'Stripe' })
 g.datasource(openai, { namespace: 'OpenAI' })
 ```
+
+### GraphQL
+
+The GraphQL connector can be created with the `GraphQL` method:
+
+```typescript
+const contentful = connector.GraphQL({
+  url: 'https://graphql.contentful.com/content/v1/spaces/{{ env.CONTENTFUL_SPACE_ID }}/environments/{{ env.CONTENTFUL_ENVIRONMENT }}',
```

Comment:
```suggestion
  url: `https://graphql.contentful.com/content/v1/spaces/${env.CONTENTFUL_SPACE_ID}/environments/${ env.CONTENTFUL_ENVIRONMENT}`,
```

Reply:
Um, does that do what we want? It takes the variable from the process environment, which can be something else than the final space where we actually need the SDL?

Reply:
https://linear.app/grafbase/issue/GB-3783/design-a-way-to-use-env-vars-in-config